lib/routes/account.js

```diff
@@ -974,6 +976,33 @@ module.exports = (log, db, mailer, Password, config, customs, checkPassword, pus
           }
         }
 
+        function sendVerifyLoginPush(devices) {
+          if (doSigninConfirmation) {
+            log.info({
+              op: 'account.signin.confirm.push.start',
+              uid: accountRecord.uid,
+              tokenVerificationId: tokenVerificationId
+            })
+
+            return push.notifyVerifyLogin(accountRecord.uid, devices)
+          }
+        }
+
+        function sendVerifyLogin() {
+          return db.devices(accountRecord.uid).then(
+            function (devices) {
```

Comment:
Prefer to use arrow functions as closures. So, `db.devices(accountRecord.uid).then(devices => { ... })`.